RealmSwift-swift2.0/ObjectSchema.swift

```diff
@@ -74,6 +74,6 @@ public final class ObjectSchema: CustomStringConvertible {
 extension ObjectSchema: Equatable {}
 
 /// Returns whether the two object schemas are equal.
-public func ==(lhs: ObjectSchema, rhs: ObjectSchema) -> Bool {
+public func == (lhs: ObjectSchema, rhs: ObjectSchema) -> Bool {
```

Comment:
It's intentional, and based on GitHub's Swift style guide: https://github.com/github/swift-style-guide#use-whitespace-around-operator-definitions